src/main/java/seedu/elisa/logic/parser/ParserUtil.java

```diff
@@ -271,9 +263,17 @@ public static LocalDateTime getFormattedDateTime(String stringDateTime) throws P
                 processedDateTime = parser.parseDateTime(stringDateTime);
                 invalidFormat = false;
                 break;
+            } catch (FastReminderParseException fp) {
+                invalidFormat = true;
+                parseException = fp;
+            } catch (MidnightParseException mp) {
+                String formatted = String.format(MESSAGE_MIDNIGHT, mp.getMessage());
+                parseException = new ParseException(formatted);
```

Comment:
I might be mistaken, but I have a feeling that there is a need to set `invalidFormat = true` in this block for it to be thrown out further down.